libraries/fabric-shim/test/unit/server.js

```diff
@@ -106,9 +100,6 @@ describe('ChaincodeServer', () => {
         it('should create a gRPC server instance with mutual TLS credentials and call addService in the constructor', () => {
```

Comment:
This test no longer checks if "addService in the constructor" is called. Not sure what it does test really.